corehq/apps/app_manager/models.py

```diff
@@ -1453,12 +1453,11 @@ def _get_media_by_language(self, media_attr, lang, strict=False):
             to return first path in sorted lang->media-path list
         """
         assert media_attr in ('media_image', 'media_audio')
-        toggle_enabled = toggles.LANGUAGE_LINKED_MULTIMEDIA.enabled
         app = self.get_app()
 
-        if self.use_default_image_for_all and media_attr == 'media_image' and toggle_enabled(app.domain):
+        if self.use_default_image_for_all and media_attr == 'media_image':
             lang = app.default_language
-        if self.use_default_audio_for_all and media_attr == 'media_audio' and toggle_enabled(app.domain):
+        if self.use_default_audio_for_all and media_attr == 'media_audio':
```

Comment:
nit, this could probably be collapsed into a single if statement.

Reply:
updated